torch/_inductor/mkldnn_lowerings.py

```diff
@@ -521,6 +533,8 @@ def qlinear_unary(
                 x_scale.realize()
             if not isinstance(x_zp, ir.TensorBox):
                 assert type(x_zp) == int
+                if x.get_dtype() == torch.int8 and x_zp == 0:
+                    sym_quantized_int8_act = True
                 x_zp = V.graph.add_tensor_constant(
                     torch.tensor(x_zp, dtype=torch.int32), name="x_zp"
```

Comment:
I am confused for DA8W8, should the `x_zp` and `x_scale` be constant? Per my understanding, it should be decided at runtime for each activation.

Reply:
`x_zp` would be constant (0) for int8 symmetric quantization.

Reply:
in case of asymmetric int8 quantization of `x`, `x_zp` may coincidentally be 0 for some specific value of the int8 activation tensor encountered in the compilation phase of `torch.compile`, but may not be zero for subsequent activation tensors encountered by the workload.

So, I'll make a change that would allow us to distinguish int8 asymmetric quantization from int8 symmetric quantization when `x_zp` would be 0.

Reply:
Hi @leslie-fang-intel and @Xia-Weiwen, I opened #143886 to discuss. Thanks!